<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>西湖景区管理系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_Main">
    <item>
     <widget class="QWidget" name="mapContainer" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_Map">
       <item>
        <widget class="QLabel" name="labelMapTitle">
         <property name="maximumSize">
          <size>
           <width>949</width>
           <height>350</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 16px; font-weight: bold; background-color: #4A6FA5; color: white; padding: 5px;</string>
         </property>
         <property name="text">
          <string>西湖景区地图</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="mapWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="controlPanel" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_Control">
       <item>
        <widget class="QGroupBox" name="groupBoxPathQuery">
         <property name="title">
          <string>路径查询</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_Path">
          <item row="0" column="0">
           <widget class="QLabel" name="labelPathStart">
            <property name="text">
             <string>起点：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBoxPathStart"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelPathEnd">
            <property name="text">
             <string>终点：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboBoxPathEnd"/>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QPushButton" name="pushButtonQueryPath">
            <property name="text">
             <string>查询最短路径</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxTour">
         <property name="title">
          <string>景点遍历</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_Tour">
          <item row="0" column="0">
           <widget class="QLabel" name="labelTourStart">
            <property name="text">
             <string>起点：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBoxTourStart"/>
          </item>
          <item row="1" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_Tour">
            <item>
             <widget class="QPushButton" name="pushButtonDFS">
              <property name="text">
               <string>DFS遍历</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonBFS">
              <property name="text">
               <string>BFS遍历</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxMST">
         <property name="title">
          <string>最小生成树</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_MST">
          <item>
           <widget class="QPushButton" name="pushButtonMST">
            <property name="text">
             <string>生成最小生成树</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonClear">
         <property name="styleSheet">
          <string notr="true">background-color: #DC143C; color: white; font-weight: bold;</string>
         </property>
         <property name="text">
          <string>清除高亮</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxResult">
         <property name="title">
          <string>查询结果</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_Result">
          <item>
           <widget class="QTextEdit" name="textEditResult">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
